languages/rust/exercises/concept/string-vs-str/.docs/hints.md

```diff
@@ -0,0 +1,12 @@
+# General references on rust strings:
+
+- [string slices discussion in TRPL book](https://doc.rust-lang.org/book/ch04-03-slices.html#string-slices)
+- [&str primitive documentation](https://doc.rust-lang.org/std/primitive.str.html)
+- [std::str](https://doc.rust-lang.org/std/str/)
+- [string types in RBE](https://doc.rust-lang.org/stable/rust-by-example/std/str.html)
+
+# Interesting string related articles:
+
+- [hermanradtke: tutorial on string versus str in functions](https://hermanradtke.com/2015/05/03/string-vs-str-in-rust-functions.html)
```

Comment:
Nit: I'd omit this link; it's both quite old and makes some factually incorrect assertions, i.e. " String is just a vector of 8-byte chars". It's not a big enough deal to hold up matters on this PR, though.